package.json

```diff
@@ -3,15 +3,15 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "firebase": "^7.14.2",
+    "firebase": "^7.14.3",
```

Comment:
One thing of note for dependency management - It's generally a good idea to indicate in your PR why you are changing the version of a dependency. This project is super young, so this shouldn't affect anything. However in any long-lived project, even a small version bump comes with risk so we should at least understand what the benefit of upgrading is.

I wouldn't worry about changing this back, but just something to note for the future!